tests/visual/grid-column-menu.html

```diff
@@ -246,7 +274,8 @@
                             </span>
                         </li>
                         <li class="k-separator k-item k-state-default" role="presentation">&nbsp;</li>
-                        <li class="k-item k-columns-item k-state-default k-last k-state-hover" role="menuitem" style="z-index: auto;">
+                        <li class="k-item k-columns-item k-state-default k-last k-state-hover" role="menuitem"
+                            style="z-index: auto;">
```

Comment:
Revert all the changes from this part:
* Formatting the code does not bring any value
* In the case of the comments, it will actually break the tests, as it will introduce whitespace